app/controllers/casa_admins_controller.rb

```diff
@@ -3,6 +3,10 @@ class CasaAdminsController < ApplicationController
   before_action :set_admin, only: [:edit, :update]
   before_action :require_organization!
 
+  def index
+    @admins = policy_scope(current_organization.casa_admins).sort_by(&:email)
```

Comment:
It would be nice if we could move sorting to the activerecord query layer (for performance reasons) but it's not a big deal, there will never be very many casa_admins per org- I just want to make sure that we encourage healthy patterns for bigger datasets

Reply:
Agreed, given that the coding effort is about the same, we might as well push for the patters we'd like to see in the codebase 👌🏼

Reply:
@stevelandwehr do you think we can sneak that change in before merging this? We can also address this before merging `new-layout` into master.